config/backend_connections.go

```diff
@@ -61,6 +76,28 @@ func (c *RethinkDBConnection) GetSession() *r.Session {
 	return c.session
 }
 
+// loadCertificate reads a certificate either from a file or from a string
+func loadCertificate() []byte {
```

Comment:
Could you add a test for this? Also, the name suggests that this is a function for loading any certificate, not just the RETHINK_TLS one. For better testing, you could implement this signature instead:

```
func importRethinkDBRootCA(opts *gorethink.ConnectOpts)
```

Additionally, if a root ca is loaded, there should be a warning using `logrus.Warnf()`, similar to https://github.com/ory-am/hydra/blob/master/cmd/host.go#L98 or https://github.com/ory-am/hydra/blob/master/cmd/host.go#L135